src/components/Locations.js

```diff
@@ -0,0 +1,33 @@
+import './Locations.css'
+import { Component } from 'react'
+
+import React from "react";
+
+class Locations extends Component{
+   constructor(){
+       super()
+       this.state={
+
+       }
```

Comment:
```suggestion
       this.state={
       locations: [],
       showLocations: false
       }
```
You should initialize your list of locations in state to begin with. Don't rely on `componentDidMount` to initialize state. You're also missing a vital piece of state which keeps track of whether to **show** or **hide** the list of locations. I added it and called it `showLocations`. It's a boolean that should toggle back and forth when you click the "Show/Hide Locations" button.